build.assets/Dockerfile

```diff
@@ -56,17 +56,33 @@ RUN git clone --depth=1 https://github.com/Yubico/libfido2.git -b 1.12.0 && \
 
 FROM buildpack-deps:18.04 AS libbpf
 
-# Install libbpf
+# Install required dependencies
 RUN apt-get update -y --fix-missing && \
     apt-get -q -y upgrade && \
     apt-get install -q -y --no-install-recommends \
         libelf-dev
 
 ARG LIBBPF_VERSION
-RUN mkdir -p /opt && cd /opt && curl -L https://github.com/gravitational/libbpf/archive/refs/tags/v${LIBBPF_VERSION}.tar.gz | tar xz && \
+RUN mkdir -p /opt && cd /opt &&  \
+    curl -L https://github.com/gravitational/libbpf/archive/refs/tags/v${LIBBPF_VERSION}.tar.gz | tar xz && \
     cd /opt/libbpf-${LIBBPF_VERSION}/src && \
+    make &&  \
+    BUILD_STATIC_ONLY=y DESTDIR=/opt/libbpf make install
+
+# Install newer libbpf.
+FROM buildpack-deps:18.04 AS libbpf-1-0-1
```

Comment:
Use a non-versioned step name, so we don't have to update it when we bump libbpf (and inevitably forget to do so, leading to a confusing step name over time).

Maybe `libbpf-teleport` and `libbpf` (or `libbpf-upstream`)?

Reply:
@codingllama My plan was to remove this step in my next PR. I don't think we should build two versions of libbpf when we don't really need them.